docs/reference/project/stack.mdx

```diff
@@ -0,0 +1,72 @@
+A nitric stack is described by its definition in a `nitric.yaml` file. A stack represents a collection of resources that operate with each other and are deployed together.
+
+## Properties
+
+| **name**    | **description**                                              | **type**                           |
+| ----------- | ------------------------------------------------------------ | ---------------------------------- |
+| name        | A descriptive name of the stack                              | `string`                           |
+| functions   | zero or more functions to be deployed                        | map of [function](./functions)     |
+| containers  | zero or more containers to be deployed                       | map of [container](./containers)   |
+| topics      | zero of more topics to be deployed                           | map of [topic](./topics)           |
```

Comment:
I think queues are missing from this list and the placeholder docs

Reply:
done